### ISSUE TITLE
[DEV] Update the Supercomputer Institute Documentation to utilize ochami

### ISSUE THREAD

**Issue by alexlovelltroy** [OpenCHAMI/roadmap#14]
LANL runs a Supercomputer Institute(SI) for interns each summer.  In the first two weeks of the session, they divide up into teams and follow our instructions to take a ten-node hpc cluster from bare metal to a functional cluster that they can use for the remainder of the summer.  The goal isn't for the teams to blindly execute a runbook.  We want them to understand each step from a manual perspective before they move on to use automation to accomplish the task.

For this summer's SI, we want to prepare enough documentation for the teams to use ochami as their main hpc management tool.  This forces us to write/revise ochami documentation without assuming any previous exposure to HPC or CSM.  In addition, it means that we'll get valuable feedback from a motivated audience.  They'll tell us what's difficult to understand or where the operations feel clunky.  We'll be able to summarize their feedback for the TSC and use it to guide other sites as they adopt ochami themselves.

**Comment by alexlovelltroy** [comment]
@synackd Now that SI24 is over, can we update the website to include a blog post about the SI practicum and release the LAUR Practicum?

**Comment by synackd** [comment]
Yes. Should we consider this done when the curriculum is published and there is a blog post?

Also, there are some quickstart changes that have occurred recently that obsolesces parts of the curriculum. For example, the `DNSMASQ_ACCESS_TOKEN` is no longer needed but is still present in version 2024.1 of the document. Should we remove this before publishing?
